eng/CodeChecks.ps1

```diff
@@ -45,14 +39,15 @@ function Invoke-Block([scriptblock]$cmd) {
 
 try {
 
-    Write-Host "Checking that solutions are up to date"
-
     Write-Host "Force .NET Welcome experience"
     Invoke-Block {
         & dotnet msbuild -version
     }
 
-    Get-ChildItem "$repoRoot/Azure.*.sln" -Recurse `
+    Write-Host "Checking that solutions are up to date"
+    Join-Path "$PSScriptRoot/../sdk" $ServiceDirectory  `
```

Comment:
Do you intend this to be required? If so, change the parameter to use the attribute property `Mandatory = $true`.

Reply:
No,  having an empty ` $ServiceDirectory` is fine.